public/app/plugins/datasource/elasticsearch/configuration/ConfigEditor.test.tsx

```diff
@@ -15,7 +15,9 @@ describe('ConfigEditor', () => {
     render(<ConfigEditor onOptionsChange={() => {}} options={createDefaultConfigOptions()} />);
 
     // Check DataSourceHttpSettings are rendered
-    expect(screen.getByRole('heading', { name: 'HTTP' })).toBeInTheDocument();
+    expect(screen.getByRole('heading', { name: 'Connection' })).toBeInTheDocument();
+    expect(screen.getByRole('heading', { name: 'Authentication' })).toBeInTheDocument();
+    expect(screen.getByRole('heading', { name: 'Advanced HTTP settings' })).toBeInTheDocument();
```

Comment:
Could we add some tests for sigv4 as well?

Reply:
I'll give it a try. For any particular reason or just for coverage? Asking because https://github.com/grafana/grafana/issues/67198

Reply:
Oh, it was because of a special logic that we have to render the sigv4. But if it isn't super straight forward to do, then we can skip it as based on https://github.com/grafana/grafana/issues/67198, we will probably remove it